charts/newrelic-k8s-metrics-adapter/values.yaml

```diff
@@ -62,18 +63,22 @@ resources:
 
 serviceAccount:
   # Specifies whether a ServiceAccount should be created for the job and the deployment.
-  create: true
+  create:
   # The name of the ServiceAccount to use.
   # If not set and create is true, a name is generated using the fullname template.
   name:
 
 # Configure podSecurityContext
 podSecurityContext:
-  enabled: false
   fsGroup: 1001
   runAsUser: 1001
   runAsGroup: 1001
```

Comment:
This overrides the one that comes from `.global`. This is the reason I has a `defaults` [helper](https://github.com/newrelic/nri-kubernetes/pull/383/commits/06f77ab4536590f58d1ea673b48365e678805742#diff-3dc1d0b38b4813f0520e3397a987711789b05548c1fc55680d8bee2f3bc18699R20-R26) and [removed from the Values](https://github.com/newrelic/nri-kubernetes/pull/383/commits/06f77ab4536590f58d1ea673b48365e678805742#diff-b1c831b81fe52267808d0d6996f50be42192cf1afbf1cd8502d6313db9ed8f67L384-L393) in the `nri-kubernetes` helm chart.

Reply:
Fixed